src/Room/TrackSearch.js

```diff
@@ -17,14 +17,20 @@ const TrackSearch = ({user, sendMessage}) => {
         }
     };
 
+    function uuid() {
```

Comment:
I think you want to generate the uuid for queueTrack on the server, not in the UI.

Reply:
actually, it should be done inside the `data_store.add_to_queue` function.  touters should not create ids, this is a job of the store.